libs/core/include/core/serializers/binary_interface.hpp

```diff
@@ -114,6 +114,9 @@ class BinaryConstructorInterface
           std::string("Cannot create container type with more than 1 << 32 elements"));
     }
 
+    // Allocating needed mempry
+    serializer_.Allocate(count);
```

Comment:
unrelated?

Reply:
It's related in that only BLS uses the binary serialiser and this solves a problem with it.